rpc/rpc.go

```diff
@@ -159,14 +159,37 @@ func (l *Link) RPC(ctx context.Context, msg *amqp.Message) (*Response, error) {
 		return nil, err
 	}
 
-	statusCode, ok := res.ApplicationProperties[statusCodeKey].(int32)
-	if !ok {
+	var statusCode int
+	statusCodeCandidates := []string{statusCodeKey, "statusCode"}
```

Comment:
It'd be nice to have consts for the status codes and description keys. Either that, or just inline the strings entirely. Seems odd to have one as a const and one as a literal.